packages/inter-protocol/src/proposals/demoIssuers.js

```diff
@@ -462,33 +462,25 @@ export const poolRates = (issuerName, record, kits, central) => {
  *   consume: { mints }
  * }} powers
  */
+
+// TODO(cth) DONOTMERGE Is this dead? I can't find references to it. It seems to
+//  do more than fund the AMM, and I don't know if it's needed somewhere.
```

Comment:
The reference was in `SIM_CHAIN_MANIFEST` above
https://github.com/Agoric/agoric-sdk/pull/7074/files#diff-892e7aad2dc384ce64eb722139fb97bf82a639fa2137ee6867141d8696348476L178

As you note, it does seem to do more than fund the AMM. It's at most a feature of the local development REPL environment. `packages/solo/test/test-home.js` should exercise it. I hope to chat with @michaelfig to jog my memory about exactly what it's supposed to do.

Reply:
It creates (demo) price authorities. We have other mechanisms for that now:
  - real oracle providers
  - a default price